static/templates/admin_user_list.handlebars

```diff
@@ -34,16 +34,13 @@
         {{/if}}
       {{/if}}
     </span>
-    {{#if is_active}}
-      <button class="button btn-primary open-user-form" title="{{t 'Edit User' }}" data-email="{{email}}">
+      <button class="button btn-primary open-user-form{{#unless is_active}} display-none{{/unless}}" title="{{t 'Edit User' }}" data-email="{{email}}">
           <i class="icon-vector-pencil"></i>
       </button>
-    {{/if}}
     <div class='admin-user-status'>
     </div>
   </td>
 </tr>
-{{#if is_active}}
 <tr class="user-name-form display-none" id="user_form_{{email}}">
```

Comment:
We did that intentionally to provide an easy way to undo after making changes by accidentally.  Of course, the original implementation didn't have a confirmation modal and so it was not hard to accidentally click the wrong thing; now that we have one, it's possible we no longer need the strikethrough approach.  What do you think?